### PR TITLE
Create a case to display mobile icon

### PR DESCRIPTION
**Pull request by dolibarr95**
Display object_phoning or  object_phoning_fax or  object_phoning_mobile png icon
I've create the 16*16 object_phoning_mobile.png file but i can't push it into dolibarr/theme/eldy/img (maybe rights access).
i will change dolibarr/core/class/commonobject.class.php ->getBannerAddress() too.

### CODE REVIEW

File: htdocs/core/lib/functions.lib.php [hunk 1]
```diff
@@ -1922,9 +1922,23 @@ function dol_print_phone($phone,$countrycode='',$cid=0,$socid=0,$addlink='',$sep
 		$titlealt=($withpicto=='fax'?$langs->trans("Fax"):$langs->trans("Phone"));
 	}
 	$rep='';
+	$picto = '';
+	if($withpicto){
+		if($withpicto=='fax'){//should we use switch case instead ?
+			$picto = 'phoning_fax';
+		}elseif($withpicto=='phone'){
```
Comment (eldy) [inline htdocs/core/lib/functions.lib.php:1929]: Why creating a new icone for standard phone ? Just keep current one

**Comment by eldy** [comment]
dir  dolibarr/theme/oneblow does not exists.
You must add icon into theme eldy and md.

File: htdocs/core/lib/functions.lib.php (hunk 1 above)
Comment (dolibarr95) [reply htdocs/core/lib/functions.lib.php:1929]: hi
I don't want to create a new one for stantard phone (in this case i use the object_phoning.png picture). I've just want to add the  object_phoning_mobile.png `$picto = 'phoning_mobile'`.

**Review by dolibarr95 — COMMENTED** [review]
Hi
I've done a error in my description text i want to add in : dolibarr/theme/eldy/img but i can't :
screencapture-github-dolibarr-dolibarr-upload-develop-htdocs-theme-eldy-img-1491201179857

Here is the file i want to add : 
object_phoning_mobile

And the svg file in a zip if you want to edit : 
[object_phoning_mobile.zip](https://github.com/Dolibarr/dolibarr/files/889151/object_phoning_mobile.zip)

tks

**Comment by eldy** [comment]
I can't see the image  object_phoning_mobile.png into the PR.

**Comment by dolibarr95** [comment]
Hi
As I said before i would like to push this png on Dolibarr but i don't know how to do it.
tks

**Comment by dolibarr95** [comment]
Hi

I can push only on my fork :
https://github.com/dolibarr95/dolibarr/blob/develop/htdocs/theme/eldy/img/object_phoning_mobile.png
Please tell me how to push here.

Tks

**Comment by dolibarr95** [comment]
Closed by error sorry

**Pull request closed without merging**